This branch doesn't work for me:

```
vstinner@mona$ ./python -m test test_traceback -j20 -F -m test_import_from_error_bad_suggestions_do_not_trigger_for_small_names -v
(...)
0:00:07 load avg: 8.01 [ 36/1] test_traceback failed (1 failure)
test_import_from_error_bad_suggestions_do_not_trigger_for_small_names (test.test_traceback.CPythonSuggestionFormattingTests.test_import_from_error_bad_suggestions_do_not_trigger_for_small_names) ... 
  test_import_from_error_bad_suggestions_do_not_trigger_for_small_names (test.test_traceback.CPythonSuggestionFormattingTests.test_import_from_error_bad_suggestions_do_not_trigger_for_small_names) (name='py') ... FAIL
test_import_from_error_bad_suggestions_do_not_trigger_for_small_names (test.test_traceback.PurePythonSuggestionFormattingTests.test_import_from_error_bad_suggestions_do_not_trigger_for_small_names) ... ok

======================================================================
FAIL: test_import_from_error_bad_suggestions_do_not_trigger_for_small_names (test.test_traceback.CPythonSuggestionFormattingTests.test_import_from_error_bad_suggestions_do_not_trigger_for_small_names) (name='py')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vstinner/python/main/Lib/test/test_traceback.py", line 3172, in test_import_from_error_bad_suggestions_do_not_trigger_for_small_names
    self.assertNotIn("vvv", actual)
AssertionError: 'vvv' unexpectedly found in "ImportError: cannot import name 'py' from 'NkosMVLvvvpdqPze' (/tmp/test_python_xfxl02nj/tmp7l231ock/NkosMVLvvvpdqPze.py)"

$ git show
commit 5612e561a96f4da735e966f527e1e01265564ae1 (HEAD -> pr/99895)
Author: Pablo Galindo <pablogsal@gmail.com>
Date:   Wed Nov 30 11:22:49 2022 +0000

    gh-99894: Ensure the local names don't collide with the test file in traceback suggestion error checking
```

--- a/Lib/test/test_traceback.py
+++ b/Lib/test/test_traceback.py
@@ -3168,7 +3168,7 @@ class SuggestionFormattingTestBase:
         for name in ("b", "v", "m", "py"):
             with self.subTest(name=name):
                 actual = self.get_import_from_suggestion(code, name)
-                self.assertNotIn("you mean", actual)
+                self.assertNotIn("Did you mean", actual)
                 self.assertNotIn("'vvv'", actual)
                 self.assertNotIn("'mom'", actual)
                 self.assertNotIn("'id'", actual)

--- a/Lib/test/test_traceback.py
+++ b/Lib/test/test_traceback.py
@@ -2978,7 +2978,7 @@ class SuggestionFormattingTestBase:
         for name in ("b", "v", "m", "py"):
             with self.subTest(name=name):
                 actual = self.get_suggestion(MyClass, name)
-                self.assertNotIn("you mean", actual)
+                self.assertNotIn("Did you mean", actual)
                 self.assertNotIn("'vvv", actual)
                 self.assertNotIn("'mom'", actual)
                 self.assertNotIn("'id'", actual)

--- a/Lib/test/test_traceback.py
+++ b/Lib/test/test_traceback.py
@@ -2979,7 +2979,7 @@ class SuggestionFormattingTestBase:
             with self.subTest(name=name):
                 actual = self.get_suggestion(MyClass, name)
                 self.assertNotIn("you mean", actual)
-                self.assertNotIn("'vvv'", actual)
+                self.assertNotIn("'vvv", actual)
                 self.assertNotIn("'mom'", actual)
                 self.assertNotIn("'id'", actual)
                 self.assertNotIn("'w'", actual)
@@ -3169,8 +3169,8 @@ class SuggestionFormattingTestBase:
             with self.subTest(name=name):
                 actual = self.get_import_from_suggestion(code, name)
                 self.assertNotIn("you mean", actual)
-                self.assertNotIn("vvv", actual)
-                self.assertNotIn("mom", actual)
+                self.assertNotIn("'vvv'", actual)
+                self.assertNotIn("'mom'", actual)
                 self.assertNotIn("'id'", actual)
                 self.assertNotIn("'w'", actual)
                 self.assertNotIn("'pytho'", actual)

